packages/wrangler/src/dev/miniflare.ts

```diff
@@ -634,23 +656,41 @@ export function buildMiniflareBindingOptions(config: MiniflareBindingsConfig): {
 		workflows: Object.fromEntries(bindings.workflows?.map(workflowEntry) ?? []),
 
 		durableObjects: Object.fromEntries([
-			...internalObjects.map(({ name, class_name }) => {
+			...internalObjects.map(({ name, class_name, script_name }) => {
 				const useSQLite = classNameToUseSQLite.get(class_name);
 				return [
 					name,
 					{
 						className: class_name,
 						useSQLite,
+						script_name,
+					},
+				];
+			}),
+			// redeclare internal durable object bindings for use by __Wrangler__LocalOnly__ExternalDurableObjectsReceiver
+			...internalObjects.map(({ class_name, script_name }) => {
+				const useSQLite = classNameToUseSQLite.get(class_name);
+				const bindingName = getIdentifier(
+					`__Wrangler__LocalOnly__DurableObject__${class_name}`
+				);
+				return [
+					bindingName,
+					{
+						className: class_name,
+						useSQLite,
+						script_name,
 					},
 				];
 			}),
```

Comment:
These bindings are also added to the user's worker and on _their_ env object but, again, I've taken care to make long+clear names for the same reasons as above